src/DOM/HTML/History.purs

```diff
@@ -0,0 +1,18 @@
+module DOM.HTML.History where
+
+import Control.Monad.Eff (Eff)
+import DOM.HTML.Types (HISTORY, History)
+import Data.Foreign (Foreign)
+import Prelude (Unit)
+
+-- DocumentTitle will set value of `document.title`
+newtype DocumentTitle = DocumentTitle String
+newtype Delta = Delta Int
+newtype URL = URL String -- Unsure how to better type this.
```

Comment:
Can you derive `Eq`, `Ord`, and `Newtype` instances for these please?

I know what you mean about `URL`, but I think it's fine as a newtype here, as the API we're providing with this library is supposed to be dumb anyway.